backend/config/corpora-api.yml

```diff
@@ -413,6 +413,39 @@ paths:
         "404":
           $ref: "#/components/responses/404"
 
+  /dp/v1/collections/index:
```

Comment:
I think `/dp/v1/collections:` could be modified to support this need. Why is a new endpoint needed?

Reply:
If we want to reuse the existing endpoint, we'd have to add parameters to specify whether to only return public and not tombstoned collections, plus add new fields (collection name and a few dates). I am not opposed to it but I think it's faster/cleaner to create a new one. Also, @MillenniumFalconMechanic , once we have the new views running, will the existing endpoint be needed at all?

Reply:
The existing `/dp/v1/collections` endpoint will still be required for the my collections functionality.